src/tex/appendix.tex

```diff
@@ -622,8 +622,21 @@ \subsection{Impact of sampling temperature}
 \end{figure}
 
 
+\clearpage
+\subsection{Summary of trends and recommendations}
+
+\oone consistently leads across both \ChemBench corpus and \chembenchmini.  \ClaudeThreeFiveSonnet, \GPTFourO, \LlamaThreeOneFourZeroFiveBInstruct form the next tier of strong performers. These models show robust performance across all skills and difficulty levels. 
+
+Larger models generally perform better (\LlamaThreeOneFourZeroFiveBInstruct > \LlamaThreeOneSeventyBInstruct > \LlamaThreeOneEightBInstruct). However, smaller but well-designed models can compete with larger models. For example, \GemmaTwoNineBIt has an overall accuracy of \variable{output/trends_section_variables/gemma_9B.txt} on \ChemBench corpus, which is only \variable{output/trends_section_variables/diff_between_llama_405B_and_gemma_9B.txt}\% less accurate than the much larger \LlamaThreeOneFourZeroFiveBInstruct model.
+
+One can observe a clear progression of performance across model families (e.g., \ClaudeThreeFiveSonnet >  \ClaudeThree > \ClaudeTwo or \GPTFourO > \GPTFour > \GPTThreeFiveTurboZeroT). Newer versions consistently outperform their predecessors.
+Scores on knowledge-intensive tasks are typically lower than those on calculation and reasoning-intensive questions. Performance on reasoning-intensive questions in \chembench tends to be the highest.
+
+Based on these findings, some scope for improving these models could be to focus on enhancing knowledge-based training, possibly through improved pre-training on chemistry-specific texts or with chemistry knowledge-base integration. However, this type of knowledge should be integrated with sound machine-learning approaches. For example, even \PaperQATwo could not outperform the \gls{llm} it is using for directing the tools, even though the agent has access to literature evidence. This suggests we might need an overhaul of how we approach agent building, and how we train models for science applications, since clearly the valuable information stored in textbooks and papers, that can be used to answer questions is not fully leveraged by any of the state-of-the-art models and systems.
```

Comment:
```suggestion
Based on these findings, some scope for improving these models could be to focus on enhancing knowledge-based training, possibly through improved pre-training on chemistry-specific texts or with chemistry knowledge-base integration. However, innovation is also needed to incorporate this information into the systems. For example, even \PaperQATwo could not outperform the \gls{llm} it is using for directing the tools, even though the agent has access to literature evidence.
This suggests we might need to not only focus on building chemistry-specific retrieval datasets since current systems fail to retrieve the relevant systems and should be coupled with more domain-specific databases. However, the observation that our ReAct agents were too fragile to answer questions for which they had custom-made tools available correctly suggests that we also must invest in building more robust agent frameworks. 
```